R/archive.R

```diff
@@ -448,6 +448,24 @@ epi_archive =
 #'   ```
 #'
 #' @export
+#' @examples 
+#' library(delphi.epidata)
+#' library(epiprocess)
+#' library(data.table)
+#' library(dplyr)
+#' 
+#' # must be given a dataframe and contain a `version` column
+#' covidcast(data_source = "jhu-csse",
+#' signals = "confirmed_7dav_incidence_prop",
+#' time_type = "day",
+#' geo_type = "state",
+#' time_value = epirange(20200601, 20200603),
+#' geo_values = "ca,fl",
+#' issues = epirange(20200601, 20200603)
+#' ) %>%
+#' fetch_tbl() %>%
+#' select(geo_value, time_value, version = issue, case_rate = value) %>%
```

Comment:
This part is all too complicated I think. Can you maybe just generate a meaningless data frame and convert that (perhaps with and without some extra keys)?